platforms/web/php/standards.md

```diff
@@ -0,0 +1,59 @@
+# PHP Standards and Style Guide
+*   [Introduction](#introduction)
+    *   [PHP](#php)
+        *   [PSR-1](#php)   
+        *   [PSR-2](#php)   
+    *   [Laravel](#laravel)
+    *   [RESTful API's](#restful)
+*   [Amendments](#amendments)
+    *   [Variables](#variables)
+    *   [Methods](#methods)
+*   [Adhering to standards](#adhering)
+    *   [PhpStorm + PHP Code Sniffer](#codesniffer)
+    *   [PhpStorm + PHP-CS-Fixer](#csfixer)
+
+
+
+<h3 id="introduction">Introduction</h3>
```

Comment:
How come you are using html tags over markdown? The majority of the file seems to be markdown?

Reply:
After wanting to build a table of contents I checked online for an example for syntax and it used header tags to enable the use of the id field for linking the contents to the headers.

Reply:
When you commit file changes in markdown, using headers, GitHub auto generate links. The annoying thing is that it is not shown in the preview while editing online

Reply:
Checkout the source for the helpers README:
https://github.com/langleyfoxall/helpers-laravel/blob/master/README.md

Here has all headers using markdown, linked and has duplicate headers so you can see how they get numbered too.